src/screens/augmentedReality/ArtworkDetailScreen.js

```diff
@@ -71,17 +59,19 @@ export const ArtworkDetailScreen = ({ route, navigation }) => {
     return <LoadingSpinner loading />;
   }
 
-  if (error || !data.length) {
+  if (!data.length) {
```

Comment:
can `data` be empty at this point? do we still need this?

Reply:
We don't really need it, but isn't security always good? 😂

Reply:
we try to do things we know and not hoping that something is ok to have or keep.

we are initializing data with `const [data, setData] = useState(route?.params?.data ?? []);`. so if there is no data from params it would be set to an empty array. this means checking for length makes sense.

it would be different, if we would not have the fallback to an empty array. in that case we would need to check what values data can have from all the possible params that can occur.